knimin/lib/data_access.py

```diff
@@ -2425,3 +2597,7 @@ def _revert_ready(self, barcodes):
                  SET results_ready = NULL
                  WHERE barcode IN %s"""
         self._con.execute(sql, [tuple(barcodes)])
+
+    def _execute_sql(self, sql, sql_args=None):
```

Comment:
I don't think this function is needed. Note that you can change all the instances of this function by:
```python
db._con.execute(sql, sql_args)
```